zeebe-cluster/templates/statefulset.yaml

```diff
@@ -51,7 +51,7 @@ spec:
         - name: ZEEBE_BROKER_THREADS_IOTHREADCOUNT
           value: {{ .Values.ioThreadCount  | quote }}
         - name: ZEEBE_BROKER_GATEWAY_ENABLE
-          value: "false"
+          value: {{ .Values.brokerEmbeddedGatewayEnable | default "false" | quote }}
```

Comment:
```suggestion
          value: {{ .Values.embeddedGatewayEnabled | default "false" | quote }}
```

Reply:
+1 @ezolnbl can you please add @npepinpe suggestions?